doc/src/conf.py

```diff
@@ -45,8 +45,10 @@
 # General information about the project.
 project = u'The Cylc Suite Engine'
 copyright = u'2008-2019 NIWA & British Crown (Met Office) & Contributors'
-version = '7.8'  # The short X.Y version.
-release = '7.8.0'  # The full version, including alpha/beta/rc tags.
+
+# Version & release set (identically) by override via 'bin/cylc-make-docs'.
+version = ''  # The short X.Y version.
```

Comment:
Rather than passing this as a command line argument we can set the version within Python:

```python
>>> from cylc.version import CYLC_VERSION
>>> CYLC_VERSION
'7.8.1'
```

Reply:
Ah, nice! This is a much cleaner approach, which I will get added in. I suspect I was grepping for ``VERSION``/``version``-related variables only in the bash of the ``tests`` dir, so missed this possibility.